docs/commercial-alternatives.rst

```diff
@@ -0,0 +1,44 @@
+
+- the commercial price
+- the number of (additional) features supplied
+- the microcontroller implementation (note that ``n/a`` in the table means that there is currently no information on this)
+
+.. note:: We do not claim that this table is complete. If you think an entry is missing, please `let us know`_ and we will add it.
+
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| Product name                  | Manufacturer                 | Microcontroller / Implementation                    | Link                                                              |
++===============================+==============================+=====================================================+===================================================================+
+| BioSemi USB trigger interface | `Biosemi`_                   | n/a                                                 | https://www.biosemi.com/faq/USB%20Trigger%20interface%20cable.htm |
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| USB TTL event marking module  | `The Black Box ToolKit Ltd`_ | n/a                                                 | https://www.blackboxtoolkit.com/usbttl.html                       |
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| Brain Products TiggerBox      | `Brain Products GmbH`_       | FTDI FT2232HL chip                                  | https://www.brainproducts.com/productdetails.php?id=55            |
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| Cedrus C-pod                  | `Cedrus Corporation`_        | PIC32MX150F128                                      | https://cedrus.com/c-pod/                                         |
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| CORTECH SOLUTIONS SD-MS-TCPUA | `Cortech Solutions Inc`_     | National Instruments digital IO device, NI USB 6501 | https://cortechsolutions.com/product/sd-ms-tcpua/                 |
++-------------------------------+------------------------------+-----------------------------------------------------+-------------------------------------------------------------------+
+| USB2TTL8 Adapter              | `LabHackers Inc`_            | MK20DX256                                           | http://labhackers.com/usb2ttl8.html                               |
```

Comment:
Link to PJRC / Teensy?

Reply:
in that list I'd only refer to "finished products" ... without assembly or anything else --> "commercial alternatives", not "list of viable microcontrollers"